### PR TITLE
Refactoring dockerfile

### PR DESCRIPTION
**Pull request by sprakash57**
This includes -

* Added Readme.md
* Removed redundant packages
* Used python 3.6 inside dockerfile
* Added gitignore file

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,13 @@
-FROM python:3.7-alpine
+FROM python:3.6
```
Comment (biswain2309) [inline Dockerfile]: Please upgrade the version of Python.

File: Dockerfile [hunk 2]
```diff
@@ -1,22 +1,13 @@
...
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
 
+WORKDIR /usr/src/app
+COPY requirements.txt ./
 RUN pip install -r requirements.txt
+COPY . .
 
-RUN python ecommerce/manage.py makemigrations
```
Comment (biswain2309) [inline Dockerfile:17]: Migrations should be a part of Docker commands

File: Dockerfile (hunk 1 above)
Comment (sprakash57) [reply Dockerfile]: Upgraded to 3.8-alpine

File: Dockerfile (hunk 2 above)
Comment (sprakash57) [reply Dockerfile:17]: Reverted.

**Review by biswain2309 — APPROVED** [review]
Looks good!

**Pull request merged**